tests/BaseTest.php

```diff
@@ -0,0 +1,104 @@
+<?php
+
+namespace Tests;
+
+use Tests\Support\AssertionException;
+use Tests\Support\AssertionExceptionExpectException;
+use Exception;
+
+class BaseTest
+{
+    public function __construct($repositoryName, $fileName)
+    {
+        $this->dirName = getenv('JSON_STORAGE_PATH');
+        $this->fileName = $fileName;
+        $this->repositoryName = $repositoryName;
```

Comment:
instead of getting this property in constructor - you can just the the current class name